DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -35,10 +35,25 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
 
             CourseContent? courseContent = null;
             return connection.Query<CourseContent, CourseSection, CourseContent>(
-                @"SELECT Customisations.CustomisationID AS id,
+                @"  WITH CustomisationDurations AS (
+                  SELECT CustomisationId,
+                         SUM(AverageDuration) AS AverageDuration
+                    FROM (SELECT Customisations.CustomisationID,
+                                 CASE
+                                 WHEN Tutorials.OverrideTutorialMins > 0 THEN Tutorials.OverrideTutorialMins
+                                 ELSE Tutorials.AverageTutMins END AS AverageDuration
+                            FROM CustomisationTutorials
+                            INNER JOIN Customisations ON CustomisationTutorials.CustomisationID = Customisations.CustomisationID
+                            INNER JOIN Tutorials ON CustomisationTutorials.TutorialID = Tutorials.TutorialID
+                            WHERE CustomisationTutorials.CustomisationID = @customisationId
+                                  AND (CustomisationTutorials.Status = 1 OR CustomisationTutorials.DiagStatus = 1)

```

Comment:
Are you sure we should include tutorials with CT.Status != 1? Doesn't look like they're included in `GetMinsForCustomisation`? 🙂

Reply:
Good point, change in 1341b0f 🙂